experiments/protox_tpch_sf10/main.sh

```diff
@@ -3,19 +3,25 @@
 set -euxo pipefail
 
 SCALE_FACTOR=10
+INTENDED_PGDATA_HARDWARE=ssd
+PGDATA_PARENT_DPATH=/mnt/nvme1n1/phw2/dbgym_tmp/
+
+# space for testing
```

Comment:
This is to run specific commands within `main.sh`. It's inconvenient to copy them directly because they rely on ENV_VARS, so this is my way of "copying" them. I'll add a comment better explaining this, but I imagine it'll be permanent.